components/shared/base/id.rs

```diff
@@ -333,6 +333,14 @@ impl WebViewId {
     pub fn installed() -> Option<WebViewId> {
         WEBVIEW_ID.with(|tls| tls.get())
     }
+
+    #[cfg(feature = "test-utils")]
+    pub fn create(namespace_id: u32, index: u32) -> WebViewId {
+        WebViewId(BrowsingContextId {
+            namespace_id: PipelineNamespaceId(namespace_id),
+            index: Index::new(index).expect("Incorrect test case"),
+        })
+    }
```

Comment:
Instead of putting this behind a feature, which could cause unnecessary recompilation, please just give it an obvious name such as `mock_for_testing`. I think the argument should be a `BrowsingContextId` as well.